plane/src/types.rs

```diff
@@ -285,6 +285,9 @@ impl ExecutorConfig {
 
 #[derive(Clone, Serialize, Deserialize, Debug)]
 pub struct SpawnConfig {
+    /// Cluster to spawn to. Uses the controller default if not provided.
+    pub cluster: Option<ClusterName>,
```

Comment:
Spawn config now takes an optional cluster name. If it is not provided, we fall back on the default cluster name set on the controller.